tests/test_pipelines/test_variant_calling.py

```diff
@@ -174,11 +205,10 @@ def test_run(self):
                                  '-T RealignerTargetCreator '
                                  '--read_filter BadCigar '
                                  '--read_filter NotPrimaryAlignment '
-                                 '-o tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id.intervals '
-                                 '-l INFO '
+                                 '-o tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id.intervals'
+                                 ' -l INFO '
                                  '-U LENIENT_VCF_PROCESSING '
-                                 '-I tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id_recal.bam '
-                                 '--known /path/to/known/indels',
+                                 '-I tests/assets/jobs/test_dataset/gatk_var_calling/test_user_sample_id_recal.bam',
```

Comment:
I was getting multiple assertion failures due to this whitespace. I do not understand why and I am not sure that moving it like I did is the best approach to solving this, but I cannot think of a better solution. Any tips welcome!